lib/notifier.js

```diff
@@ -26,43 +22,90 @@ var SETTINGS = {
   branch: null,  // git branch name
   notifier: {
     name: 'node_rollbar',
-    version: exports.VERSION
+    version: packageJson.version
   },
   scrubHeaders: [],
   scrubFields: ['passwd', 'password', 'secret', 'confirm_password', 'password_confirmation'],
   addRequestData: null  // Can be set by the user or will default to addRequestData defined below
 };
 
+/*
+ * Public API
+ */
 
-var apiClient;
+var Notifier = function(api, options) {
+  var opt;
 
+  this.accessToken = api.accessToken;
+  this.apiClient = api;
+  this.settings = JSON.parse(JSON.stringify(SETTINGS));
+  this.VERSION = SETTINGS.notifier.version;
```

Comment:
Is `this.VERSION` used anywhere? If it is, this should probably be `this.version = this.settings.notifier.version;`